clients/unshaded/src/main/java/tachyon/client/lineage/AbstractLineageClient.java

```diff
@@ -46,16 +47,20 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   protected LineageContext mContext;
 
+  public AbstractLineageClient() {
```

Comment:
Guys, I have pushed this three liner in this PR, to address [TACHYON-1096].  The fix is small, that's why I chose not to open another PR. Let me know if you prefer that instead.